magento/pwa-studio
# Remove button clipping

## Description

Recently, we allowed a change to reach `develop`, which causes buttons to clip their text when it overflows. That's not actually the behavior we want (clipped text is useless on a button), and it has some unfortunate side effects. Text wrapping will be handled soon in an upcoming PR, but for now, we're removing the clipping.

## Related Issue
PWA-693

## Acceptance 

### Verification Stakeholders
- @dpatil-magento 
- @sirugh 

### Specification

### Verification Steps


1. Open any view with a button.
2. Verify that no part of the text is clipped.

## Screenshots / Screen Captures (if appropriate)

## Checklist
* I have updated the documentation accordingly, if necessary.
